docs/contribution/35-guidelines/20-privacy.mdx

```diff
@@ -0,0 +1,110 @@
+---
+title: Privacy and GDPR Policy for Extensions in the mStudio marketplace
+sidebar_label: Privacy policy
+---
+
+This privacy and GDPR policy establishes the requirements that contributors must follow to ensure compliance with data protection regulations, including GDPR, when developing and distributing extensions in the mStudio Marketplace. The goal is to protect user privacy and maintain transparency in data handling.
+
+## Data Deletion Policy
+
+### Deletion of Data on Uninstallation
+
+When a user uninstalls an _Extension Instance_, any data stored or processed by the extension related to that extension instance must be securely deleted. This includes:
+
+- User-generated data stored locally or remotely by the extension.
+- Logs or backups created by the extension during its operation.
```

Comment:
Might be unfeasible in some cases (for example, most extensions will manage all instances in a single deployment unit, meaning that backups and logs will contain data from all instances). Maybe change the wording to something like "should be asserted that backups and logs are deleted _eventually_"...? 🤔